app/controllers/qsets_controller.rb

```diff
@@ -38,8 +38,12 @@ def show
         @filter_mine = true
         @questions = @questions.where(user_id: current_user)
       end
+      # display question-only view
+      if @qset.settings(:permissions).qset_type == 'questions'
```

Comment:
I've just realized that `qset_type` isn't really a 'permission' per se, is it? It's more like a generic qset-level setting, than a permission. We may want to consider moving it into a new `.settings(:key)` at some point for clarity, but for now it works and is at least consistent with the other qset-level settings -- just mentioning it to document it somewhere.